config/studio/content-types/page/entry/config.xml

```diff
@@ -10,4 +10,9 @@
 	<quickCreatePath />
 	<noThumbnail>false</noThumbnail>
 	<image-thumbnail>home.png</image-thumbnail>
+	<paths>
+    <excludes>
+      <pattern>^/site/.*</pattern>
+    </excludes>
```

Comment:
What's this for? 
Format file please.

Reply:
This will prevent the create content dialog to show the homepage (as requested in ticket)